server/services/database/documents.js

```diff
@@ -44,5 +79,9 @@ const getUserDocuments = userId => {
 
 module.exports = {
   getAllDocuments,
+api-endpoint
+  createDocument
+
   getUserDocuments
+ master
```

Comment:
Same with this, I don't think this should be written.

Reply:
it shouldnt be there,i removed it